main.tf

```diff
@@ -36,3 +38,23 @@ resource "aws_iam_group" "users" {
 }
 
 # Attach users policy(policies.tf)
+
+resource "aws_iam_policy" "users-default" {
+  name_prefix = "default-users-policy"
+  path        = "/users/"
+  description = "Default policy for a user to self manage"
+  policy      = data.aws_iam_policy_document.users.json
+}
+
+
+resource "aws_iam_policy" "admins-assume-admin" {
+  name_prefix = "admins-assume-admin"
+  description = "Allows to assume role in another AWS account"
+  policy      = data.aws_iam_policy_document.assume-admin.json
+
+}
+
+resource "aws_iam_group_policy_attachment" "this" {
```

Comment:
And this one is `admins-assume-admin`, but the attachment should be under the group in `groups.tf`